src/sourmash/signature.py

```diff
@@ -192,6 +193,16 @@ def __reduce__(self):
             ),
         )
 
+    def __copy__(self):
+        a = SourmashSignature(
+            mh=self.minhash,
+            self.name,
+            filename=self.filename,
+        )
+        a.merge(self)
```

Comment:
this is probably not necessary; it should be redundant, since `a` already contains all the information in `self` due to passing in `self.minhash`.